docs/contributor/greenboot.md

```diff
@@ -46,25 +46,31 @@ sudo journalctl -o cat -u greenboot-healthcheck.service
 
 ### Health Check Implementation
 
-The script utilizes the MicroShift health check functions that are available
-in the `/usr/share/microshift/functions/greenboot.sh` file to reuse procedures
-already implemented for the MicroShift core services. These functions need a
-definition of the user workload namespaces and the expected count of pods.
-
-```bash
-PODS_NS_LIST=(busybox)
-PODS_CT_LIST=(1      )
-```
+The script utilizes the MicroShift `healthcheck` command that is part of the
+`microshift` binary to reuse procedures already implemented for the MicroShift
+core services. 
 
 The script starts by running sanity checks to verify that it is executed from
-the `root` account and that the MicroShift service is enabled.
-
-Finally, the MicroShift health check functions are called to perform the
-following actions:
-- Get a wait timeout of the current boot cycle for the `wait_for` function
-- Call the `namespace_images_downloaded` function to wait until pod images are available
-- Call the `namespace_pods_ready` function to wait until pods are ready
-- Call the `namespace_pods_not_restarting` function to verify pods are not restarting
+the `root` account.
+
+Then, executes `microshift healthcheck` command with following options:
+- `-v=2` to increase verbosity of the output
+- `--timeout="${WAIT_TIMEOUT_SECS}s"` to override default 300s timeout value
```

Comment:
Is 300s the default in the code, or 600?

Reply:
It's 300s:
https://github.com/openshift/microshift/blob/b7bde43ebf1d0a2b0d650f43bd66546b4d0924ce/pkg/cmd/healthcheck.go#L67